.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: node_js
+
+os: linux
+dist: bionic
+
 node_js:
   - 10.15.3
-  - node
+  - lts/*
 
+cache: npm
+
+before_install:
+  - npm install -g npm@latest
 install:
   - npm install
 
 script:
   - npm test
-
-matrix:
-  allow_failures:
```

Comment:
I think we should keep the allow-failures on the node version that are not production mandatory, ie lts/*